src/assets/lang/en/metatags-descriptions.json

```diff
@@ -363,5 +363,10 @@
     "id": "elections-campaign",
     "title": "Make Internxt Great Again",
     "description": "Internxt's Elections Campaign 2024"
+  },
+  {
+    "id": "monitor",
```

Comment:
Use a better id. Remember to change it for all languages.

Reply:
Done